src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -712,6 +712,46 @@ def test_sql_db_operation_mgmt(self, resource_group, resource_group_location, se
                  .format(resource_group, server, database_name, ops[0]['name']))
 
 
+class SqlServerDbShortTermRetentionScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    @SqlServerPreparer()
+    @AllowLargeResponse()
+    def test_sql_db_short_term_retention(self, resource_group, server):
```

Comment:
you need to record for the test with yaml file provided. otherwise CI will fail.

Reply:
SqlServerDbShortTermRetentionScenarioTest doesn't pass yet locally. Should I still upload its yaml file? Looks like yaml file is auto generated after run the scenario test.

Reply:
> https://github.com/Azure/azure-sdk-for-python/blob/release/v3/sdk/sql/azure-mgmt-sql/azure/mgmt/sql/operations/_managed_databases_operations.py#L40 You bump the dependency, which means you change all commands already. Since you're using a new sdk using a new api-version. You need to re-run these tests lively.

I added new swagger shortTermRetentionPolicies.json to V2020-02-02-Preivew. 
The managedDatabases.json is not I added. But looks like since it is a new swagger in version V2020-02-02-Preivew so it was forced updated in new SDK. 

I might not familiar with other swagger's scenario tests. Is there any faster way to make it work?